requests.JSONDecodeError missing from types-requests

While `requests.exceptions.JSONDecodeError` was added in 032e6ee90cbb938259a2aa2183966502de19108e in types-requests, that covers only one of the two exposed instances. It was also imported into the top level of requests (https://github.com/psf/requests/commit/db575eeedcfdb03bf31285afd3033e301df8b685, `requests/__init__.py`), which is not reflected in the stubs here. Using `requests.JSONDecodeError` in code will cause mypy to complain:
> error: Module has no attribute "JSONDecodeError"  [attr-defined]

while code using this definition works fine. Changing that line to use `requests.exceptions.JSONDecodeError` makes the mypy error go away, while not changing anything in the execution except that the line itself is longer. 

It looks like it should be imported between these two lines:
https://github.com/python/typeshed/blob/749d3db815926eeeb4705897c18cd0641af55cc7/stubs/requests/requests/__init__.pyi#L18-L19

--- a/stubs/requests/requests/__init__.pyi
+++ b/stubs/requests/requests/__init__.pyi
@@ -16,6 +16,7 @@ from .exceptions import (
     ConnectTimeout as ConnectTimeout,
     FileModeWarning as FileModeWarning,
     HTTPError as HTTPError,
+    JSONDecodeError as JSONDecodeError,
     ReadTimeout as ReadTimeout,
     RequestException as RequestException,
     Timeout as Timeout,

